Makefile

```diff
@@ -113,3 +114,14 @@ gendocs: docs/minikube.md
 docs/minikube.md: $(shell find cmd) $(shell find pkg/minikube/constants) pkg/minikube/cluster/assets.go
 	$(MKGOPATH)
 	cd $(GOPATH)/src/$(REPOPATH) && go run -ldflags="$(K8S_VERSION_LDFLAGS) $(MINIKUBE_LDFLAGS)" -tags gendocs gen_help_text.go
+
+.PHONY: deb
+deb: 
```

Comment:
Also you could avoid the PHONY tag and name this rule:

out/minikube_$(DEB_VERSION).deb